simple_app/templates/base.html

```diff
@@ -92,7 +92,7 @@
 
     <!-- Aladin -->
     <link rel="stylesheet" href="//aladin.u-strasbg.fr/AladinLite/api/v2/latest/aladin.min.css" />
-
+    {{ resources }}
```

Comment:
Could this the line that is writing `<bokeh.resources.Resources object at 0x7fcfbdbece50>` at the top of each page?

Reply:
Yes it is, I had thought it was required but apparently it is not.